assets/javascripts/discourse/templates/modal/assign-user.hbs

```diff
@@ -25,8 +25,21 @@
         </a>
       {{/each}}
     </div>
-    <label>{{i18n "discourse_assign.assign_modal.note_label"}}</label>
-    {{textarea id="assign-modal-note" value=model.note key-down=(action "handleTextAreaKeydown")}}
+    {{#if this.statusEnabled}}
+      <div class="control-group assign-status">
+        <label>{{i18n "discourse_assign.assign_modal.status_label"}}</label>
+        {{combo-box
+          id="assign-status"
+          content=availiableStatuses
+          value=model.status
```

Comment:
This means that the default value of the dropdown is empty: 

<img width="442" alt="image" src="https://user-images.githubusercontent.com/368961/182650615-ebbbf500-dc1e-4595-aa86-478ea8db5378.png">

where in fact it should be the default status (i.e. New), right?

Reply:
We are setting the first status as a default in the pre-validation model hook too, but I guess it would be better UX to set it as default in the UI too?